ShippoTesting/CustomsDeclarationTest.cs

```diff
@@ -30,6 +30,17 @@ public void testValidRetrieve()
         }
 
         [Test]
+        public void testValidRetrieveWithAddressImporter ()
+        {
+            CustomsDeclaration testObject = CustomsDeclarationTest.getDefaultObject1();
+            CustomsDeclaration retrievedObject;
+
+            retrievedObject = apiResource.RetrieveCustomsDeclaration ((string)testObject.ObjectId);
+            Assert.AreEqual (testObject.ObjectId, retrievedObject.ObjectId);
+            Assert.IsNotNull (retrievedObject.AddressImporter);
```

Comment:
Can we also be more specific here? Unless this is just a string, then this should be fine